scripts/build-homepage

```diff
@@ -22,8 +38,14 @@ cp -ra /usr/local/share/elektra/tool_data /usr/local/share/elektra/tool_data_bac
 # if tests were ok, we can install
 make install
 
+
+# use installed binaries and libs
+export PATH=${INSTALL_PATH}/bin:${PATH}
+export LD_LIBRARY_PATH=${INSTALL_PATH}/lib
```

Comment:
Why use a non-standard path in the first place?

Reply:
We now have a elektra installation in place (because of the puppet setup). I thought it would make more sense to put it in a complete separate place then. But I'm not really sure if it is working this way ;)

Reply:
I see, yes, the separate installation makes sense then.

> But I'm not really sure if it is working this way ;)

Yes, it should work without troubles. Without RPATH you need to extend the `LD_LIBRARY_PATH` to include the plugins. But as you use `make install` this should not be necessary, though.